### PR TITLE
Add support for writing rest and epytext docstrings

### PR DESCRIPTION
**Pull request by cbillingham**

### CODE REVIEW

**Comment by cbillingham** [comment]
@AWhetter, I broke backwards compatibility in the configuration with this change, but it seemed cleaner to consolidate the custom output configuration as they all ended up being the same. I'll bump this up to v2.0.0 when I release, unless you have objections.

File: doc/source/intro.rst [hunk 1]
```diff
@@ -12,36 +12,34 @@ Docconvert can be installed through pip:
 
     pip install docconvert
 
-Usage
+Usagegit
```
Comment (AWhetter) [inline doc/source/intro.rst]: Is this a typo?

File: src/docconvert/writer/rest.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        Args:
+            element (tuple): The docstring element.
+        """
+        args, keywords = [], []
+        for arg in self.doc.arg_fields.values():
+            if self.config.output.separate_keywords and arg.optional:
+                keywords.append(arg)
+            else:
+                args.append(arg)
+        if args:
```
Comment (AWhetter) [inline src/docconvert/writer/rest.py]: Trivial: `args` will always be a list so we can remove this `if` and let the for loop handle it.
Similarly for `keywords` below.

File: tests/test_core.py [hunk 3]
```diff
@@ -63,65 +65,35 @@ def test_file_ext(self):
...
+            ("rest", "google"),
+            ("rest", "numpy"),
+            ("rest", "epytext"),
+            ("rest", "rest"),
+            ("epytext", "google"),
+            ("epytext", "numpy"),
+            ("epytext", "epytext"),
+            ("epytext", "rest"),
+        ],
+    )
```
Comment (AWhetter) [inline tests/test_core.py:80]: Nice! :D

**Review by AWhetter — APPROVED** [review]

**Comment by AWhetter** [comment]
Doing a major version bump sounds like the right thing to do to me!

File: doc/source/intro.rst (hunk 1 above)
Comment (cbillingham) [reply doc/source/intro.rst]: yep 😞

File: src/docconvert/writer/rest.py (hunk 2 above)
Comment (cbillingham) [reply src/docconvert/writer/rest.py]: duh. thank you!

**Pull request merged**